oidc-primer.html

```diff
@@ -0,0 +1,1219 @@
+extract information such as the user’s WebId. A client will also exchange the ID token for
+an access token at an authorization server.</p>
+    <li data-md>
+     <p><code>"refresh_token":	"eyJhbGciOiJ..."</code>: The refresh token. The client will use this to fetch a
+new ID token when the current token expires.</p>
+   </ul>
+   <h3 class="heading settled" data-level="4.2" id="request-flow"><span class="secno">4.2. </span><span class="content">Request Flow</span><a class="self-link" href="#request-flow"></a></h3>
+   <p><img src="oidc-primer-request.svg" width="980"></p>
+   <p>In this example, Alice has logged into <code>https://decentphotos.example</code> and has completed the
+authentication steps above. She wants to make a request to Bob’s Storage to get a photo album
+information at <code>https://bob.otherpod.example/private/photo_album.ttl</code>. Bob has previously
+granted access to Alice but has not granted access to anyone else.</p>
+   <h4 class="no-num heading settled" id="request-flow-step-1"><span class="content">1. Discover Authorization Server</span><a class="self-link" href="#request-flow-step-1"></a></h4>
+   <p>Client can discover Authorization Server by making request to the resource</p>
+   <p>Request:</p>
+<pre class="language-http highlight">GET https://bob.otherpod.example/private/photo_album.ttl
+</pre>
+   <p>Response:</p>
+<pre class="language-http highlight"><c- kr>HTTP</c-><c- o>/</c-><c- m>1.1</c-> <c- m>401</c-> <c- ne>Unauthorized</c->
+<c- e>WWW-Authenticate</c-><c- o>:</c-> <c- l>UMA realm="example",</c->
+  <c- l>as_uri="https://auth.otherpod.example",</c->
+  <c- l>ticket="016f84e8-f9b9-11e0-bd6f-0021cc6004de"</c->
+</pre>
+   <h4 class="no-num heading settled" id="request-flow-step-2"><span class="content">2. Request AS configuration</span><a class="self-link" href="#request-flow-step-2"></a></h4>
+   <p>Knowing Authorization Server now client can discover its Token Endpoint (<a data-link-type="biblio" href="#biblio-rfc8414">[RFC8414]</a>)</p>
+   <p>Request:</p>
+<pre class="language-http highlight">GET https://auth.otherpod.example/.well-known/uma2-configuration
+</pre>
+   <p>Response:</p>
+<pre class="language-json highlight"><c- p>{</c->
+    <c- f>"issuer"</c-><c- p>:</c-> <c- u>"https://auth.otherpod.example"</c-><c- p>,</c->
+    <c- f>"token_endpoint"</c-><c- p>:</c-> <c- u>"https://auth.otherpod.example/token"</c-><c- p>,</c->
+    <c- f>"grant_types_supported"</c-><c- p>:</c-> <c- p>[</c->
+        <c- u>"urn:ietf:params:oauth:grant-type:uma-ticket"</c->
+    <c- p>]</c->
+    ...
+<c- p>}</c->
+</pre>
+   <h4 class="no-num heading settled" id="request-flow-step-3"><span class="content">3. Creates a DPoP header token</span><a class="self-link" href="#request-flow-step-3"></a></h4>
+   <p>Before we request access token, we need to generate a DPoP header token. A new DPoP token must be
+generated every time a request is made.</p>
```

Comment:
As a stand-alone comment, that you need to generate a DPoP token for every request is confusing to me. When the request is made with an Access Token, that wouldn't trigger DPoP token generation, would it? As I said, I'm not terribly used to these protocols, but you are welcome to use me as a guinea pig for what others may find confusing. :-)